src/pages/Registration/Steps/getRegistrationState.ts

```diff
@@ -122,7 +122,7 @@ function formatDocumentation({
     isKYCRequired: KycDonorsOnly ? "Yes" : "No",
 
     //general
-    activeInCountries: ActiveInCountries,
+    activeInCountries: ActiveInCountries.map((c) => ({ label: c, value: c })),
```

Comment:
`type Documentation` is already primed to be used  as `FormValue`, aws shape should be transformed here `hqCountry` , `activityCountries`